plugin/dnssec/handler.go

```diff
@@ -55,13 +55,6 @@ var (
 		Help:      "The number of elements in the dnssec cache.",
 	}, []string{"server", "type"})
 
-	cacheCapacity = prometheus.NewGaugeVec(prometheus.GaugeOpts{
```

Comment:
this needs to be removed from the README as well. (no-one ever complained this wasn't exported, so I'm OK with removing it)

Reply:
Thanks.

Reply:
@miekg 
I can not find it in README,
Can you help me find out it?
Thank you!

Reply:
ah, it's not there. my mistake